### PR TITLE
feat: filter button component in explorer

### PR DESCRIPTION
**Pull request by jake-bassett**

### CODE REVIEW

File: projects/observability/src/shared/dashboard/data/graphql/table/entity/entity-table-data-source.model.ts [hunk 1]
```diff
@@ -38,6 +38,10 @@ export class EntityTableDataSourceModel extends TableDataSourceModel {
   })
   public childEntityType?: ObservabilityEntityType;
 
+  public getScope(): string {
```
Comment (aaron-steinfeld) [inline projects/observability/src/shared/dashboard/data/graphql/table/entity/entity-table-data-source.model.ts:41]: can this take in a row ? hacky, but don't see another way.

Comment (jake-bassett) [reply projects/observability/src/shared/dashboard/data/graphql/table/entity/entity-table-data-source.model.ts:41]: A row? Do you mean array?

Comment (aaron-steinfeld) [reply projects/observability/src/shared/dashboard/data/graphql/table/entity/entity-table-data-source.model.ts:41]: For entities (and other trees), we may have rows in the table that have a different scope from other rows. I don't remember where this is called, but the data source itself doesn't have enough information to give a scope without being told what row it's giving the scope for.

Comment (anandtiwary) [reply projects/observability/src/shared/dashboard/data/graphql/table/entity/entity-table-data-source.model.ts:41]: Service list?

Comment (anandtiwary) [reply projects/observability/src/shared/dashboard/data/graphql/table/entity/entity-table-data-source.model.ts:41]: Can we address the review comments in a separate PR? I kind of pulled on latest htc and now need those changes to continue t-ui changes.

Comment (jake-bassett) [reply projects/observability/src/shared/dashboard/data/graphql/table/entity/entity-table-data-source.model.ts:41]: I created a story for the backlog.

**Pull request merged**